packages/api/schema/stryker-core.json

```diff
@@ -237,7 +237,7 @@
       "default": []
     },
     "concurrency": {
-      "description": "Set the concurrency of workers. Stryker will always run checkers and test runners in parallel by creating worker processes (note, not `worker_threads`). This defaults to n-1 where n is the number of cpu's available on your machine. This is a sane default for most use cases.",
+      "description": "Set the concurrency of workers. Stryker will always run checkers and test runners in parallel by creating worker processes (note, not `worker_threads`). This defaults to n-1 where n is the number of cpu's available on your machine, unless CPU count is <= 4, in that case it uses the CPU count. This is a sane default for most use cases.",
```

Comment:
> the number of cpu's available

'number of CPU threads' would be more accurate. Calling them 'cores' or 'logical cores' works too.

Reply:
Fixed ✅

Good find